index.html

```diff
@@ -1909,36 +1909,53 @@ <h3>Defining 'string'</h3>
 <p>[[[#char_def]]].</p>
 </div>
 
-<p>Specifications need to be clear about the encoding and processing of textual data. The recommendations in this section are mutually consistent with those in [[DESIGN-PRINCIPLES]]. In general, specifications should only support well-formed Unicode code point strings and should avoid the use of (or access to) the underlying code units or the use of different character encodings.</p>
+<p>Strings are a fundamental component of the protocols and document formats that make up the Web. Because there are multiple ways of working with strings, different terminology has evolved to support the needs of different specifications. Be sure to understand your specification's needs and use the most appropriate and precise terminology.</p>
+
+<p>The best practices found in this section are mutually consistent with those in [[DESIGN-PRINCIPLES]]. The definitions in this section use terms found in the <cite>Internationalization Glossary</cite> [[I18N-GLOSSARY]]. Some of these definitions are taken directly from [[WEBIDL]], [[INFRA]], or the Unicode glossary; in which case the definitions are quoted verbatim and include links to their source. Please refer to instructions in the Internationalization Glossary for how to import and link definitions in your own specification.</p>
+
+<p>[[UNICODE]] forms the basis for text on the Web, so the basic definition of a string is: <em>a well-formed sequence of zero or more <a>Unicode Scalar Values</a></em>. Because the term 'string' is ambiguous, this definition can be more specifically referred to using one of these definitions:
```

Comment:
And CSS specifications can refer to the definition in https://www.w3.org/TR/css-values-4/#strings ?

Reply:
3629 is the (IETF) definition of UTF-8, not string.

CSS's definition is weird, since it appears to be a USVString but points to Infra's (DOMString) definition of string. 

I don't think our set of references needs to be exhaustive and fewer references is actually better for our customers.